server/storage/files/S3Storage.ts

```diff
@@ -13,14 +25,14 @@ export default class S3Storage extends BaseStorage {
   constructor() {
     super();
 
-    this.client = new AWS.S3({
-      s3BucketEndpoint: env.AWS_S3_ACCELERATE_URL ? true : undefined,
-      s3ForcePathStyle: env.AWS_S3_FORCE_PATH_STYLE,
-      accessKeyId: env.AWS_ACCESS_KEY_ID,
-      secretAccessKey: env.AWS_SECRET_ACCESS_KEY,
+    this.client = new S3Client({
+      forcePathStyle: env.AWS_S3_FORCE_PATH_STYLE,
+      credentials: {
+        accessKeyId: env.AWS_ACCESS_KEY_ID || "",
+        secretAccessKey: env.AWS_SECRET_ACCESS_KEY || "",
+      },
       region: env.AWS_REGION,
       endpoint: this.getEndpoint(),
-      signatureVersion: "v4",
```

Comment:
This condition is deprecated and now is replaced by `import "@aws-sdk/signature-v4-crt";` according to https://github.com/aws/aws-sdk-js-v3#functionality-requiring-aws-common-runtime-crt